x-pack/plugins/security_solution/public/app/deep_links/index.ts

```diff
@@ -377,15 +376,10 @@ export const securitySolutionsDeepLinks: SecuritySolutionDeepLink[] = [
         ],
       },
       {
-        id: SecurityPageName.threatIntelligence,
-        title: THREAT_INTELLIGENCE,
-        path: THREAT_INTELLIGENCE_PATH,
-        navLinkStatus: AppNavLinkStatus.hidden,
-        keywords: [
-          i18n.translate('xpack.securitySolution.search.threatIntelligence', {
-            defaultMessage: 'Threat Intelligence',
-          }),
-        ],
+        ...getSecuritySolutionDeepLink<SecurityPageName>('indicators'),
```

Comment:
most of the values are now coming from the Threat Intelligence plugin directly, only what seemed to be more Security Solution related stays here

Reply:
sorry my first comment wasn't clear, I meant things like the features and order properties (it didn't make sense to me to have the 9006 value within the Threat Intelligence plugin), but also the navLinkStatus (as the AppNavLinkStatus lives within the Security Solution plugin).

We could move all of them inside the Threat Intelligence plugin, but these last 3 felt forced... happy to change this if you feel like they should!